bc/channel/apps.py

```diff
@@ -6,4 +6,4 @@ class UsersConfig(AppConfig):
 
     def ready(self):
         # Implicitly connect a signal handlers decorated with @receiver.
-        from bc.channel import signals
+        from bc.channel import signals  # noqa: F401
```

Comment:
Ruff's [F401 / unused-import rule](https://docs.astral.sh/ruff/rules/unused-import/) removes unused imports with the `--fix` option. This is generally desirable, but in the case of such implicit-action imports, we need to ignore the rule.